Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Shotguns/mou53_shotgun.yml

```diff
@@ -68,3 +68,5 @@
       rmc-aslot-rail: -0.02, 0.15
       rmc-aslot-stock: -0.65, -0.18
       rmc-aslot-underbarrel: 0.5, -0.25
+  - type: RMCFireGroup
+    delay: 4
```

Comment:
you can just remove the component alltogther, Breech loader inherits from base shotgun so it should be fine